### PR TITLE
Update pipeline script to find new Dockerfile

### PR DESCRIPTION
**Pull request by SueChaplain**
New dockerfile now located in /buildenv directory.
Pipeline script amended to build from this location.

Closes: #154

Signed-off-by: Sue Chaplain <sue_chaplain@uk.ibm.com>

### CODE REVIEW

**Comment by SueChaplain** [comment]
@AdamBrousseau - please can you check the change to the script?

File: JenkinsFile_build_container.groovy [hunk 1]
```diff
@@ -40,7 +40,9 @@ timeout(time: 6, unit: 'HOURS') {
                         // Tag Regular build Containers with BUILD_NUMBER and 'latest'
                         TAGS = "-t ${NAMESPACE}/${CONTAINER_NAME}:${BUILD_NUMBER} -t ${NAMESPACE}/${CONTAINER_NAME}:latest"
                     }
-                    sh "docker build -f Dockerfile ${TAGS} ."
+                    dir('buildenv') {
+						sh "docker build -f Dockerfile ${TAGS} ."
+						}
```
Comment (AdamBrousseau) [inline JenkinsFile_build_container.groovy]: The change looks good. Just some formatting differences. It looks like you've used tabs on line 44,45. The rest of the file uses spaces. Switching back should resolve the appearance above.

**Comment by AdamBrousseau** [comment]
Jenkins build container

**Comment by SueChaplain** [comment]
Closing. Changes done in PR 156

**Pull request closed without merging**